delegator/src/sign/sign.service.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Test, TestingModule } from '@nestjs/testing'
+import { SignService } from './sign.service'
+import { PrismaService } from '../prisma.service'
+import Caver, { AbiItem } from 'caver-js'
+import { dummyFactory } from './helper/dummyFactory'
+import { SignDto } from './dto/sign.dto'
+
+const PROVIDER_URL = 'https://api.baobab.klaytn.net:8651'
+const caver = new Caver(PROVIDER_URL)
+const keyring = caver.wallet.keyring.createFromPrivateKey(process.env.CAVER_PRIVATE_KEY)
```

Comment:
https://github.com/Bisonai/orakl/issues/378
I have created new issue for this problem. Previously, I we put private key for testing `caver-js` on `core` directory. How about to use same private key? Since both are only for testing on `baobab`.